mimic/test/test_resource.py

```diff
@@ -284,4 +283,4 @@ def test_verbose_logging(self):
         self.assertNotEqual(None, response_match)
         self.assertEqual(url, response_match.group('url'))
         headers = json.loads(response_match.group('headers'))
-        self.assertEqual(['application/json'], headers.get('Content-Type'))
+        self.assertEqual(['close'], headers.get('Connection'))
```

Comment:
Weird. It consistently fails on py27, pypy, and py34 if I put it back. I can't imagine how this ever got set before.  Will investigate.